Content.Server/Atmos/Piping/Binary/EntitySystems/GasVolumePumpSystem.cs

```diff
@@ -105,7 +105,7 @@ private void OnVolumePumpUpdated(EntityUid uid, GasVolumePumpComponent pump, Atm
                 return;
 
             // We multiply the transfer rate in L/s by the seconds passed since the last process to get the liters.
-            var removed = inlet.Air.RemoveVolume(pump.TransferRate * args.dt);
```

Comment:
It seems like it'd be better to just create a variant of AtmosDeviceUpdateEvent for pumps and such, and then just scale the time there. Or maybe just scale the time globally for all atmos devices?

Reply:
This is a good suggestion. I'm going to take a look through the code again and make sure there isn't any "special" code, e.g. that doesn't depend on `dt`. If there's nothing "special", I will make this change.

Reply:
I'd like to point out that this mess also exists in part because instead of using `Atmospherics.PumpGasTo()`, pumped devices copy/paste code because they need a slightly different local variant with access to intermediate variables.

Unless you think otherwise, I think that should be a separate refactoring change.